src/source/Ice/IceAgent.h

```diff
@@ -188,7 +188,7 @@ struct __IceAgent {
     CHAR localPassword[MAX_ICE_CONFIG_CREDENTIAL_LEN + 1];
     CHAR remoteUsername[MAX_ICE_CONFIG_USER_NAME_LEN + 1];
     CHAR remotePassword[MAX_ICE_CONFIG_CREDENTIAL_LEN + 1];
-    CHAR combinedUserName[MAX_ICE_CONFIG_USER_NAME_LEN + 1];
+    CHAR combinedUserName[(MAX_ICE_CONFIG_USER_NAME_LEN+1)<<1];//!< the combination of remote user name and local user name.
```

Comment:
This will fail the clang format. Try

CHAR combinedUserName[(MAX_ICE_CONFIG_USER_NAME_LEN + 1) << 1];

Or, better yet

CHAR combinedUserName[2 * MAX_ICE_CONFIG_USER_NAME_LEN + 1];

Reply:
I am not sure what you mean? I saw all the checks have passed. I did use CHAR combinedUserName[(MAX_ICE_CONFIG_USER_NAME_LEN+1)<<1]; and [2 * MAX_ICE_CONFIG_USER_NAME_LEN + 1] can not fix the warning since it has ":" between remote and local name. Did I miss something?

Reply:
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/src/source/Ice/IceAgent.c#L497

Reply:
The issue with the first approach is the spacing around the + and << which would cause a formatting issue.

Reply:
That's weird. According to the travis' result, 1675.1 clang-format Check did not fail.

Reply:
Indeed it's weird. I am seeing a lot of errors in the Travis CI run for this: https://travis-ci.com/github/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/jobs/393015520

Could you do me a favor, could you please run this locally and see if this passes? I will also look at it tomorrow morning

Reply:
let me try it later since the clang shell scripts did not work on my desktop. I am trying to resolve the clang-format.

Reply:
> Indeed it's weird. I am seeing a lot of errors in the Travis CI run for this: https://travis-ci.com/github/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/jobs/393015520
> 
> Could you do me a favor, could you please run this locally and see if this passes? I will also look at it tomorrow morning

Those errors are exactly the same as what I got so that's why I asked which specific version of clang-format does this sdk prefer.

Reply:
For my local, it did not work and the following log is as below. 
yc@yc-Inspiron-3470:~/_workspace/ycyang1229/amazon-kinesis-video-streams-webrtc-sdk-c$ ./clang-format.sh -f src/source/Ice/IceAgent.h
Clang formatting the files specified
YAML:12:32: error: invalid boolean
AllowShortBlocksOnASingleLine: Never
                               ^~~~~
Error reading /home/yc/_workspace/ycyang1229/amazon-kinesis-video-streams-webrtc-sdk-c/.clang-format: Invalid argument
yc@yc-Inspiron-3470:~/_workspace/ycyang1229/amazon-kinesis-video-streams-webrtc-sdk-c$

Reply:
FYI
https://releases.llvm.org/6.0.1/tools/clang/docs/ClangFormatStyleOptions.html
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
For my Mac, it is fine since its version is 10.0.1
YongChengs-MacBook-Pro:amazon-kinesis-video-streams-webrtc-sdk-c yc.yang$ ./clang-format.sh -f src/source/Ice/IceAgent.h
Clang formatting the files specified
YongChengs-MacBook-Pro:amazon-kinesis-video-streams-webrtc-sdk-c yc.yang$ git stat
git: 'stat' is not a git command. See 'git --help'.

The most similar commands are
	status
	stage
	stash
YongChengs-MacBook-Pro:amazon-kinesis-video-streams-webrtc-sdk-c yc.yang$ git status
On branch fix-warning-for-combined-name
Your branch is up to date with 'origin/fix-warning-for-combined-name'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/source/Ice/IceAgent.h

no changes added to commit (use "git add" and/or "git commit -a")
YongChengs-MacBook-Pro:amazon-kinesis-video-streams-webrtc-sdk-c yc.yang$ git diff
diff --git a/src/source/Ice/IceAgent.h b/src/source/Ice/IceAgent.h
index e3391b65..af2d5f1c 100644
--- a/src/source/Ice/IceAgent.h
+++ b/src/source/Ice/IceAgent.h
@@ -188,7 +188,7 @@ struct __IceAgent {
     CHAR localPassword[MAX_ICE_CONFIG_CREDENTIAL_LEN + 1];
     CHAR remoteUsername[MAX_ICE_CONFIG_USER_NAME_LEN + 1];
     CHAR remotePassword[MAX_ICE_CONFIG_CREDENTIAL_LEN + 1];
-    CHAR combinedUserName[(MAX_ICE_CONFIG_USER_NAME_LEN+1)<<1];//!< the combination of remote user name and local user name.
+    CHAR combinedUserName[(MAX_ICE_CONFIG_USER_NAME_LEN + 1) << 1]; //!< the combination of remote user name and local user name.
 
     RtcIceServerDiagnostics rtcIceServerDiagnostics[MAX_ICE_SERVERS_COUNT];
     RtcIceCandidateDiagnostics rtcSelectedLocalIceCandidateDiagnostics;
YongChengs-MacBook-Pro:amazon-kinesis-video-streams-webrtc-sdk-c yc.yang$ clang-format --version
clang-format version 10.0.1 
YongChengs-MacBook-Pro:amazon-kinesis-video-streams-webrtc-sdk-c yc.yang$